models/product.model.lkml

```diff
@@ -12,3 +13,20 @@ explore: fct_active_users {
     sql_on: ${fct_active_users.daily_server_id} = ${dim_daily_server_info.daily_server_id} ;;
   }
 }
+
+explore: fct_active_servers {
+  label: "Telemetry Active Servers"
+  group_label: " [New] Active Servers"
+
+  join: dim_version {
```

Comment:
would it make sense to eventually use this in the existing views (e.g. `dim_daily_server_info.view`?

Reply:
Not sure I get the suggestion. Is it to split daily server info dimensions in two parts, with one being the version and the other the server information?

Reply:
Just asking if we can reuse these "seed" dimensions somehow. If not, so it be, would be nice though if we could - easier to maintain.